pom.xml

```diff
@@ -59,12 +59,12 @@
   <ciManagement />
   <distributionManagement>
     <snapshotRepository>
-      <id>cloudbees-nexus-snapshots</id>
+      <id>sonatype-nexus-snapshots</id>
```

Comment:
:-1:

Projects using cloudbees-oss-parent to deploy should be deploying using the cloudbees username/password as that is the only one approved to deploy into `com.cloudbees` and related group ids.

Thus you should be using the shared username/password. Contact me if you don't have it